x-pack/plugins/ml/public/application/components/scatterplot_matrix/scatterplot_matrix_vega_lite_spec.ts

```diff
@@ -209,7 +217,12 @@ export const getScatterplotMatrixVegaLiteSpec = (
               },
             },
           }
-        : {}),
+        : {
+            selection: {
```

Comment:
Is it possible to get this same interaction on `click` e.g. I just want an alternative way to select a single doc?

Reply:
Added in 2265c5308d9cf789f1e285e8b54273d5c7b7b31d